src/vmm_mad/remotes/lib/vcenter_driver/importer.rb

```diff
@@ -801,32 +801,12 @@ def self.import_networks(con_ops, options)
                 ar_str << ",SIZE = \"#{size}\"]"
 
                 n[:one] << ar_str
+                n[:clusters][:one_object] = n[:one]
 
-                one_vn = VCenterDriver::VIHelper.new_one_item(OpenNebula::VirtualNetwork)
-                if n[:clusters] #Distributed Port Group
-                    done = []
-                    for i in 0..n[:clusters][:refs].size-1
-                        cl_id = n[:clusters][:one_ids][i]
-                        next if cl_id == -1 || done.include?(cl_id)
+                one_vn = VCenterDriver::Network.create_one_network(n[:clusters])
```

Comment:
object passed to create_one_network should be explicitly created to improve readibility